unstructured/partition/text.py

```diff
@@ -257,6 +257,7 @@ def element_from_text(
     text: str,
     coordinates: Optional[tuple[tuple[float, float], ...]] = None,
     coordinate_system: Optional[CoordinateSystem] = None,
+    languages: Optional[list[str]] = None,
```

Comment:
This kwarg was in some partitioning functions but not all of them before, correct? @scanny would you rather go with `**kwargs` here?

Reply:
I think this is fine for the time being. This argument is omissible, like `coordinates` and `coordinate_system` are, so if the caller doesn't have it they can just leave it out of the call.

But I think all the partitioners have this parameter explicitly defined because it's used in the `apply_lang_metadata()` call which I believe is global to all partitioners.